providers/redis/shedlock-provider-redis-quarkus2/src/test/java/net/javacrumbs/shedlock/provider/redis/quarkus/QuarkusRedisLockProviderIntegrationTest.java

```diff
@@ -0,0 +1,103 @@
+package net.javacrumbs.shedlock.provider.redis.quarkus;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
+
+import javax.inject.Inject;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.redis.datasource.RedisDataSource;
+import io.quarkus.test.junit.QuarkusTest;
+
+@QuarkusTest
+public class QuarkusRedisLockProviderIntegrationTest {
```

Comment:
For test please extend `AbstractExtensibleLockProviderIntegrationTest` like [here](https://github.com/lukas-krecan/ShedLock/blob/master/providers/redis/shedlock-provider-redis-jedis4/src/test/java/net/javacrumbs/shedlock/provider/redis/jedis4/JedisLockProviderIntegrationTest.java) (maybe without the nestin for cluster and pool)

Reply:
Done